agent-packaging/packaging-scripts/90-google-osconfig-agent.preset

```diff
@@ -0,0 +1 @@
+enable google-osconfig-agent.service
```

Comment:
this file already exists in root of repo; why are you adding a new one instead of moving?
why rename packaging/ to agent-packaging/ ?

Reply:
this came from my initial idea where i wanted to keep the packaging for all agents in a single place and not just osconfig-agent. I should have reverted it, since we not using that model. will revert it